projects/SSTorytime/services/sstorytime/tests/basic.nix

```diff
@@ -0,0 +1,80 @@
+{
+  pkgs,
+  sources,
+  ...
+}:
+
+{
+  name = "SSTorytime basic test";
+
+  nodes = {
+    machine =
+      { config, pkgs, ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.sstorytime
+          sources.modules.services.sstorytime
+          sources.examples.SSTorytime."Enable SSTorytime programs"
+          sources.examples.SSTorytime."Enable SSTorytime server"
+        ];
+      };
+  };
+
+  # Debug interactively with:
+  # - nix run .#checks.x86_64-linux.projects/SSTorytime/nixos/tests/basic.driverInteractive -L
+  # - run_tests()
+  interactive.sshBackdoor.enable = true; # ssh -o User=root vsock/3
+  interactive.nodes = {
+    machine =
+      {
+        lib,
+        config,
+        ...
+      }:
+      {
+        # forward ports from VM to host
+        virtualisation.forwardPorts = [
+          {
+            from = "host";
+            host.port = config.services.sstorytime.port;
+            guest.port = config.services.sstorytime.port;
+          }
+        ];
+
+        # forwarded ports need to be accessible
+        networking.firewall.enable = false;
+      };
+  };
+
+  testScript =
+    { nodes, ... }:
+    let
+      cfg = nodes.machine.services.sstorytime;
+    in
+    # python
+    ''
+      start_all()
+
+      machine.wait_for_unit("postgresql.target")
```

Comment:
Postgresql is required for `sstorytime.service`, let's test without explicitly waiting for it.
```suggestion
```